MantidQt/CustomInterfaces/src/Reflectometry/QtReflRunsTabView.cpp

```diff
@@ -46,13 +46,13 @@ void QtReflMainView::initLayout() {
   ui.splitterTables->setStretchFactor(0, 0);
   ui.splitterTables->setStretchFactor(1, 1);
 
+  // Create the DataProcessor presenter
   ReflGenericDataProcessorPresenterFactory presenterFactory;
+  auto processorPresenter = presenterFactory.create();
```

Comment:
This temporary `processorPresenter` is not required. I would suggest invoking `presenterFactory.create()` at line 54 directly into the constructor of the `QDataProcessorWidget`. 

Secondly, and it won't matter if you fix this above by avoiding the temporary, but don't use processorPresenter.release(), when you could use `std::move`